docs/DeveloperGuide.md

```diff
@@ -416,17 +419,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+| `* * *`  | efficient insurance agent                  | find a client by name                                   | locate details of clients without having to go through the entire list |
+| `* * *`  | insurance agent                            | edit a client's information                             | update client details upon changes and add new information in existing client       |
+| `* * *`  | opportunistic insurance agent              | filter my clients by their existing insurance plans     | upsell a new, similar plan to them                                     |
+| `* * *`  | popular insurance agent                    | view all my appointments without browsing the whole app | get a quick reminder of all my appointments                            |
+| `* * *`  | busy insurance agent                       | schedule my meetings with my contacts                   | remember when I am meeting my clients                                  |
+| `* * *`  | time-constrained insurance agent           | view my appointments in a chronological list            | schedule my day efficiently according to the meetings I have           |            |
+| `* * *`  | ambitious insurance agent                  | mark my meeting as completed                            | mark my successful progress in selling insurance to clients            |
+| `* * *`  | insurance agent                            | edit my meeting's details                               | update meeting details upon changes                                    |
+| `* * *`  | accommodating insurance agent              | relocate my meeting's location                          | change the location of my meeting if client request for it             |
+| `* * *`  | accommodating insurance agent              | reschedule my meeting to another date                   | change my meeting to another date if client cannot make it on the specified date|
+| `* * *`  | insurance agent                            | delete a meeting                                        | cancel meetings that I no longer need                                  |
+| `* * *`  | busy insurance agent                       | find a meeting by client                                | locate details of meetings without having to go through the entire list|                                 |
+| `* * *`  | time-constrained insurance agent           | find a meeting by date                                  | list the meetings I have on a specific date                            |                                 |
+| `* * `   | active insurance agent                     | track my clients’ insurance expiry                      | remind my clients to renew their contract                              |
+| `* *`    | insurance agent with many contacts         | save my client’s likes/dislikes                         | interact with them more effectively and increase the chance of insurance sale |                          |
```

Comment:
Not sure if the testers will mistake saving of likes/dislikes as a feature we were going to do